.github/codeowners

```diff
@@ -1,7 +1,7 @@
 # Matched against repo root (asterisk)
-# *                         @mishraomp @DerekRoberts @LocalNewsTV @fergmag
+* @LocalNewsTV @fergmag @kcaparas1630 @Shakespeares-spy @GDamaso
```

Comment:
Why'd you remove me? I'm still participating during startup. 

cc: @fergmac

Reply:
Also, Fergus' username was misspelled.

Reply:
You were removed due to a miscommunication during handoff on Friday, thank you for catching the Fergus typo.
Previously the codeowners was commented out/non-enforced, which misled the assumption it was just part of the template.